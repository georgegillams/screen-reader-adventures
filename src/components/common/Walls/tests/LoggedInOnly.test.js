import React from 'react';
import { render } from '@testing-library/react';

import { LoggedInOnly } from '..';

describe('<LoggedInOnly />', () => {
  it('Should render correctly', () => {
    const { container } = render(<LoggedInOnly user={null}>Test</LoggedInOnly>);

    expect(container).toMatchSnapshot();
  });
  it('Should render correctly with user', () => {
    const { container } = render(<LoggedInOnly user={{}}>Test</LoggedInOnly>);

    expect(container).toMatchSnapshot();
  });
});
