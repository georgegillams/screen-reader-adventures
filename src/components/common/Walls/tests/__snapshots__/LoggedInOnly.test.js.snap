// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoggedInOnly /> Should render correctly 1`] = `
<div>
  <div
    class="section__outer"
  >
    <h2
      class="section__heading"
      id="logged-out"
    >
      Logged out
    </h2>
    <span
      class="paragraph"
    >
      You need to be logged in to 
      view this content
      <br />
      <a
        class="text-link text-link--text-link text-link--no-padding text-link--inline"
        href="/sign-up"
      >
        Register here - it's quick and easy.
      </a>
      <br />
      <a
        class="text-link text-link--text-link text-link--no-padding text-link--inline"
        href="/login"
      >
        Already got an account? Log in here.
      </a>
    </span>
  </div>
</div>
`;

exports[`<LoggedInOnly /> Should render correctly with user 1`] = `
<div>
  Test
</div>
`;
